### ISSUE TITLE
Use webpack

### ISSUE THREAD

**Issue by yogiben** [cheersjosh/corona#3]
Based on the technologies used, I imagine that the kind of people who like this stack would be interested in using webpack.

Is this on the horizon?

**Comment by cheersjosh** [comment]
@yogiben Yeah definitely, I love webpack. I'm tied up launching a project before next Friday, but plan to work on this repo after that.

**Comment by lyquocnam** [comment]
+1 webpack, currently, i am using webpack with meteor 1.3 rc 12. all good !

**Comment by lyquocnam** [comment]
hey, don't abandon this project man... this awesome ! no long to see u men
